### PR TITLE
Small tweaks to GenesisTxValidator and a basic test class

### PR DESCRIPTION
**Pull request by chirhonul**
See commit messages for more details. This work is not particularly drastic / likely useful, but I wanted to send it out earlier rather than later for some feedback.

### CODE REVIEW

**Comment by chirhonul** [comment]
I pushed another commit b148592 to try to follow the style guide in terms of ordering of import statements. I would be happy to squash the commits if desired after review (before merging).

File: src/test/java/bisq/core/dao/node/validation/GenesisTxValidatorTest.java [hunk 1]
```diff
@@ -61,7 +61,7 @@ public void testGetGenesisTx() {
         // With mismatch in blockheight and tx id, we should not get genesis tx back.
         Optional<Tx> result = GenesisTxValidator.getGenesisTx(genesisTxId, genesisBlockHeight, genesisTotalSupply, rawTx);
         Optional<Tx> want = Optional.empty();
-        assertEquals(want, result);
+        Assert.assertEquals(want, result);
```
Comment (ManfredKarrer) [inline src/test/java/bisq/core/dao/node/validation/GenesisTxValidatorTest.java]: We don't have a defined code style guide for static imports but usually if repeated calls we use static imports, specially if they are well known like Assert.

File: src/test/java/bisq/core/dao/node/validation/GenesisTxValidatorTest.java [hunk 2]
```diff
@@ -41,11 +41,11 @@ public void testGetGenesisTx() {
         String blockHash = "abc123";
         Coin genesisTotalSupply = Coin.parseCoin("2.5");
         long time = new Date().getTime();
-        final List<TxInput> inputs = asList(
+        final List<TxInput> inputs = Arrays.asList(
```
Comment (ManfredKarrer) [inline src/test/java/bisq/core/dao/node/validation/GenesisTxValidatorTest.java]: In that case I personally also prefer to not use the static import to not obscure the information where asList is defined.

**Review by ManfredKarrer — APPROVED** [review]
utACK

File: src/test/java/bisq/core/dao/node/validation/GenesisTxValidatorTest.java [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+        result = GenesisTxValidator.getGenesisTx(genesisTxId, genesisBlockHeight, genesisTotalSupply, rawTx);
+        Tx tx = new Tx(rawTx);
+        tx.setTxType(TxType.GENESIS);
+        for (int i = 0; i < tx.getTxOutputs().size(); ++i) {
+            tx.getTxOutputs().get(i).setTxOutputType(TxOutputType.GENESIS_OUTPUT);
+        }
+        want = Optional.of(tx);
+        assertEquals(want, result);
+        // TODO(chirhonul): test that only outputs in tx summing exactly to genesisTotalSupply is accepted, and
+        // that code under test raises RuntimeError otherwise.
```
Comment (ManfredKarrer) [inline src/test/java/bisq/core/dao/node/validation/GenesisTxValidatorTest.java]: Maybe we should add a custom Exception there (extending RuntimeError)? Would make test more clear as well if u can check exception type.

**Pull request merged**